source/javascripts/app/_lang.js

```diff
@@ -143,11 +143,8 @@ under the License.
   // returns a new query string with the new language in it
   function generateNewQueryString(language) {
     var url = parseURL(location.search);
-    if (url.language) {
```

Comment:
I think it was for a more cosmetically appealing URL. Basically it just allowed for `?videojs` instead of `?language=videojs`